### DISCUSSION TITLE
TAGs at Kubecon

### DISCUSSION THREAD

**Discussion by joshgav** [cncf/toc#1206] [Ideas]
In the same vein of updates from Kubernetes SIGs, it would be nice to provide an update from CNCF TAGs about new and updated projects in their domains as part of Kubecon keynotes. IMO, as Kubernetes SIGs are to the Kubernetes project, CNCF TAGs are to the broader cloud-native ecosystem.

Relatedly, each TAG should probably hold a broad project meeting at Kubecon to bring together their many sandbox projects for awareness and discussions. Should Kubecon's tracks themselves be aligned with TAGs?

**Comment by TheFoxAtWork** [comment]
We would welcome the TAGs to provide opinions here so we can share with the events team.  @nikhita has been doing a lot in this space so there may be an opportunity to expand more.

Potentially target recommended improvements for North America and share with the events team - ideally by end of Feb.
